.github/workflows/backend.yml

```diff
@@ -103,7 +106,8 @@ jobs:
       - uses: actions/download-artifact@v2
         name: Download Binary Package
         with:
-          name: binary-package
+          # Only run cluster test on jdk8
```

Comment:
Add an `if` condition to this step otherwise the job will be run twice on jdk 8

Reply:
I find this part is not under `build` so it will not run twice?

Reply:
> I find this part is not under `build` so it will not run twice?

Oh right, my fault, the matrix part was collapsed in GitHub website